lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -3252,48 +3248,6 @@ def option_values_target_ports
     return res
   end
 
-  def cmd_go_pro_help
-    print_line "Usage: go_pro"
-    print_line
-    print_line "Launch the Metasploit web GUI"
-    print_line
-  end
-
-  def cmd_go_pro(*args)
-    @@go_pro_opts.parse(args) do |opt, idx, val|
-      case opt
-      when "-h"
-        cmd_go_pro_help
-        return false
-      end
-    end
-    unless is_apt
-      print_warning "This command is only available on deb package installations, such as Kali Linux."
-      return false
-    end
-    unless is_metasploit_debian_package_installed
-      print_warning "You need to install the 'metasploit' package first."
-      print_warning "Type 'apt-get install -y metasploit' to do this now, then exit"
-      print_warning "and restart msfconsole to try again."
-      return false
-    end
-    # If I've gotten this far, I know that this is apt-installed, the
-    # metasploit package is here, and I'm ready to rock.
-    if is_metasploit_service_running
```

Comment:
I don't see any other reference to the method `is_metasploit_service_running`. Maybe worths to delete de method too?